linux/CMakeLists.txt

```diff
@@ -1,11 +1,9 @@
 #Copyright (C) Microsoft Corporation. All rights reserved.
 
 set(pal_common_h_files
-    ../common/inc/azure_c_pal/gballoc.h
 )
 
 set(pal_common_c_files
-    ../common/src/gballoc.c

```

Comment:
>gballoc [](start = 18, length = 7)

So is gballoc/mocking still working in the Linux unit tests?

Reply:
Good question :). Errrr... let me... errr.... WSL!

---
In reply to: [462450700](https://github.com/Azure/azure-c-pal/pull/21#discussion_r462450700) [](ancestors = 462450700)

Reply:
Keyword was "still" I claim innocence! I didn't change anything! Likely it is "still" working, if it "ever" worked...



---
In reply to: [463131355](https://github.com/Azure/azure-c-pal/pull/21#discussion_r463131355) [](ancestors = 463131355,462450700)